index.html

```diff
@@ -236,5 +236,42 @@ <h3>About Myself</h3>
         </li>
       </ul>
     </section>
+    <section class="contact">
+      <div class="check">
+        <h4>contact me</h4>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it!
+        </p>
+        <form action="https://formspree.io/f/xbjeaboy" method="POST">
+          <ul>
+            <li>
+              <input
+                type="text"
+                name="user_name"
+                maxlength="30"
+                placeholder="Full name"
+              />
+            </li>
+            <li>
+              <input type="email" name="user_email" placeholder="enter email" />
+            </li>
+            <li>
+              <textarea
+                name="user_message"
+                placeholder="write your message here..."
+                maxlength="500"
+                rows="4"
+                cols="50"
+              ></textarea>
+            </li>
+            <li class="button">
+              <button type="submit">Get in Touch</button>
```

Comment:
- Hey, please, let's kindly improve the design on our button by removing the black border added, add the interaction to the button to indicate the `hover, active, enabled, disabled` state to improve the user experience when they interact with the button.


![Screenshot from 2023-03-28 14-25-13](https://user-images.githubusercontent.com/59397159/228252059-2b377771-fb21-4200-9bf7-d237696dee9e.png)
![Screenshot from 2023-03-28 14-24-26](https://user-images.githubusercontent.com/59397159/228252063-168e9be7-fc52-4753-a169-e28faca642ba.png)

Reply:
- [ ] Kindly implement the button interactions properly according to the design as suggested earlier; when we reproduce a given design perfectly, it shows that we pay attention to details. 👍 

## Figma design
<img width="282" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/228341456-12e8b219-d025-4f8a-a09d-16a5deb8f45e.png">